dashboard/app/controllers/api/v1/census/census_controller.rb

```diff
@@ -0,0 +1,108 @@
+class Api::V1::Census::CensusController < ApplicationController
+  include SchoolInfoDeduplicator
+  skip_before_action :verify_authenticity_token
+
+  CENSUS_FIELDS = [
+    :school_year,
+    :submitter_email_address,
+    :submitter_name,
+    :submitter_role,
+    :how_many_do_hoc,
+    :how_many_after_school,
+    :how_many_10_hours,
+    :how_many_20_hours,
+    :other_classes_under_20_hours,
+    :topic_blocks,
+    :topic_text,
+    :topic_robots,
+    :topic_internet,
+    :topic_security,
+    :topic_data,
+    :topic_web_design,
+    :topic_game_design,
+    :topic_other,
+    :topic_other_description,
+    :topic_do_not_know,
+    :class_frequency,
+    :tell_us_more,
+    :pledged
+  ].freeze
+
+  BOOLEAN_FIELDS = [
+    :other_classes_under_20_hours,
+    :topic_blocks,
+    :topic_text,
+    :topic_robots,
+    :topic_internet,
+    :topic_security,
+    :topic_data,
+    :topic_web_design,
+    :topic_game_design,
+    :topic_other,
+    :topic_do_not_know,
+    :pledged
+  ].freeze
+
+  # POST /dashboardapi/v1/census/<form_version>
+  def new_submission
```

Comment:
nit: Any reason we're not using the [standard](http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) `create` action?

Reply:
Just lack of awareness on my part. I'll make this change.